File: Easy/LongestCommonPrefix/SolutionTest.java
package Easy.LongestCommonPrefix;

import org.junit.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class SolutionTest{

    @Test
    public void longestCommonPrefixTest(){
        assertEquals("fl", new Solution().longestCommonPrefix(new String[]{"flower", "flow", "flight"}));
    }

    @Test
    public void longestCommonPrefixTest2(){
        assertEquals("", new Solution().longestCommonPrefix(new String[]{"dog", "racecar", "car"}));
    }
}

File: Easy/LongestCommonPrefix/SolutionTest.py
from unittest import TestCase, main
from Solution import Solution

class SolutionTest(TestCase):
    def test_longestCommonPrefix(self):
        self.assertEqual("fl", Solution().longestCommonPrefix(["flower", "flow", "flight"]))


    def test_longestCommonPrefix2(self):
        self.assertEqual("", Solution().longestCommonPrefix(["dog", "racecar", "car"]))


if __name__ == "__main__":
    main()
